test/GithubApi.Repositories.test.js

```diff
@@ -0,0 +1,58 @@
+const { StatusCodes } = require('http-status-codes');
+const { expect } = require('chai');
+const axios = require('axios');
+const chai = require('chai');
+const chaiSubset = require('chai-subset');
+const md5 = require('md5');
+
+chai.use(chaiSubset);
+
+const urlrepos = 'https://api.github.com/repos/aperdomob/jasmine-json-report';
+
+describe('Repositories GET', () => {
+  let userResponse;
+
+  before(async () => {
+    userResponse = await axios.get('https://api.github.com/users/aperdomob');
+  });
+  it('Consume GET Service on GITHUB API', async () => {
+    expect(userResponse.status).to.equal(StatusCodes.OK);
+    expect(userResponse.data.name).to.equal('Alejandro Perdomo');
+    expect(userResponse.data.company).to.equal('Perficient Latam');
+    expect(userResponse.data.location).to.equal('Colombia');
+  });
+
+  describe('Using hypermedia', () => {
+    let jasmineRepo;
+    before(async () => {
+      const repositoriesResponse = await axios.get(`${userResponse.repos_url}`);
+      jasmineRepo = repositoriesResponse.data.find((repo) => repo.name === 'jasmine-json-report');
+    });
+    it('Finding by Hypermedia', async () => {
```

Comment:
the `async` keyword is not needed here, the `async-await` is just needed when you are using axios or something that returns a `promise`

Reply:
I erased it, and the error still appears